common/add_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
+
+Understanding the `AddOperationError` exception.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .warning }
+> _An **Add** operation targeting an existing profile could end up effectively performing an **Alter** operation on this profile. Pyracf will always raise an `AddOperationError` to bring attention to these failures before attempting the **Add** operation._
+
+&nbsp;
+
+When pyracf executes an **Add** operation, it first performs an **Extract** to evaluate whether the profile already exists. If the **Return Code** and the **Messages** returned by the **Extract** operation indicate that the profile already exists, an `AddOperationError` will be raised to indicate that the request failed. An `AddOperationError` can be handled as follows.
```

Comment:
```markdown
When pyracf executes an **Add** operation, it first attempts a **Profile Extract** to evaluate whether the profile already exists. If the **Return Code** and the **Messages** returned by the **Profile Extract** operation indicate that the profile already exists, an `AddOperationError` will be raised and the requested **Add** operation will not be executed. An `AddOperationError` can be handled as follows.
```

Reply:
Sure, agreed that this is better. Changed.